### PR TITLE
Pf styles

### PR DESCRIPTION
**Pull request by andrii-kuskovets**

### CODE REVIEW

File: app/scss/blocks/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    &__secondary {
+        border-radius: 31px;
+        background-color: rgba($white, .1);
+        font-size: 16px; //19
+        font-weight: 400;
+        padding: 10px 15px;
+    }
+}
```
Comment (ladomiryak) [inline app/scss/blocks/_buttons.scss:34]: would be better to move all the values into some variable and map it. 
e.g.:
```
const buttons = {
  variants: {
    primary: {
      bg: '#000',
      fs: '14px',
      color: '#000',
    },
    secondary: {
      ...
    },
  },
  sizes: {
    xs: {
      ...
    }
  }
}
```

File: app/scss/blocks/_about.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__img {
+        margin-bottom: $mb-mobile;
+    }
+
+    &__title {
+        @include back-heading('About');
```
Comment (ladomiryak) [inline app/scss/blocks/_about.scss:12]: its not flexible. get value from html attribute. 

`content: attr(data-heading) " ";`

https://developer.mozilla.org/ru/docs/Web/CSS/attr

File: app/scss/blocks/_menu.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            text-decoration: underline;
+        }
+    }
+}
+
+.hamburger {
+    display: block;
+    background-color: transparent;
+    padding: 0;
+    border: none;
```
Comment (ladomiryak) [inline app/scss/blocks/_menu.scss:49]: you can move it into `.btn-reset` for example.
`border: 0` is correct to use here.

File: app/scss/blocks/_sessions.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+        &__img {
+            display: flex;
+            margin-bottom: $mb-mobile;
+        
+            img {
+                width: 85px;
+            }
+
+            img+img {
```
Comment (ladomiryak) [inline app/scss/blocks/_sessions.scss:58]: `& + & {...` will do the same..

File: app/scss/blocks/_sessions.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+                margin-left: 25px;
```
Comment (ladomiryak) [inline app/scss/blocks/_sessions.scss:59]: put it inside `img` and use` & + & {..`

File: app/scss/style.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+@import 'blocks/mailing';
+@import 'blocks/form';
+@import 'blocks/projects';
+@import 'blocks/schedule';
+@import 'blocks/footer';
+
+* {
+    box-sizing: border-box;
+
+    img {
```
Comment (ladomiryak) [inline app/scss/style.scss:29]: move it from` * {`

File: app/scss/blocks/_about.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    }
+
+    &__hide {
```
Comment (BearSwan) [inline app/scss/blocks/_about.scss:15]: move it from `.about` context to global.
Create global class smth like `d-none`, so you can use it overall site.

File: app/scss/blocks/_advantages.scss [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    &__subtitle {
+        margin-bottom: $mb-mobile;
+    }
+
+    &__subtext {
+        text-align: center;
+        opacity: 0.49;
+        font-size: 12px;
```
Comment (BearSwan) [inline app/scss/blocks/_advantages.scss:37]: I've noticed that font-size 12px is used in 3 cases.
Please create a varriable

File: app/scss/blocks/_faq.scss [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+    &__subtitle {
+        &_active {
+            margin-bottom: $mb-mobile;
+        }
+    }
+
+    &__paragraph {
+        display: none;
+        font-size: 14px;
```
Comment (BearSwan) [inline app/scss/blocks/_faq.scss:20]: same here
move 14px to variable

File: app/scss/blocks/_mailing.scss [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    &__img {
+        width: 275px;
+        margin: 0 auto;
+        position: relative;
+
+        &:after {
+            content: '';
+            background-image: url(../img/Union3.png);
```
Comment (BearSwan) [inline app/scss/blocks/_mailing.scss:27]: combine all background properties in one line, like: 
`background: bg-color bg-image position/bg-size bg-repeat bg-origin bg-clip bg-attachment initial|inherit;`

**Comment by ladomiryak** [comment]
> Please make the following changes:
> 
> 1. Add hover state and check cursor for buttons and tabs


its mobile layout. you can skip first point for now, I guess...

**Pull request merged**